### PR TITLE
[9.0] hr_public_holidays allow duplicate of year

### PR DESCRIPTION
**Pull request by Martronic-SA**
As it is now, this is not possible to duplicate a year that makes it boring to creates new years.
That PR allows to easily create new years by making the copy of the year and related lines plus one year.
It allows also to deletion the created year if needed.

This will have to be forwarded to 10.0 and 11.0

### CODE REVIEW

File: hr_public_holidays/models/hr_public_holidays.py [hunk 1]
```diff
@@ -28,6 +29,7 @@ class HrPublicHolidays(models.Model):
         'hr.holidays.public.line',
         'year_id',
         'Holiday Dates'
+        copy=True
```
Comment (gurneyalex) [inline hr_public_holidays/models/hr_public_holidays.py]: missing comma on line 31

**Review by gurneyalex — CHANGES_REQUESTED** [review]

File: hr_public_holidays/models/hr_public_holidays.py (hunk 1 above)
Comment (Martronic-SA) [reply hr_public_holidays/models/hr_public_holidays.py]: oh yes, thank you!

**Comment by Martronic-SA** [comment]
The travis errors are not coming from my PR but underlying in the base public holidays module.
I just want to merge the copy part of my PR, I don't want to review the whole module.
What can I do?

**Comment by feketemihai** [comment]
@Martronic-SA Can you please fix travis errors, even that is not coming from your PR, tere are just 2 small details. Thanks.

**Comment by feketemihai** [comment]
@Martronic-SA I changed the file online to pass flake and pylint...hope to be ok...

**Comment by feketemihai** [comment]
@Martronic-SA Can you add a simple test for copy...otherwise :+1:

**Review by feketemihai — CHANGES_REQUESTED** [review]
Please add some tests

**Pull request closed without merging**